pyslack/__init__.py

```diff
@@ -41,6 +41,22 @@ def chat_post_message(self, channel, text, **params):
         })
         return self._make_request(method, params)
 
+    def chat_update_message(self, channel, text, timestamp, **params):
+        """chat.update
+
+        This method updates a message.
```

Comment:
So after reading the docs, I see that `ts` has to be the timestamp of the original message sent - can you make a comment here specifying that?